.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ defaults: &defaults
 
     - run:
         name: Verify screenshots and generate diffs and new screenshots
-        command: node dockers/Screenshotter/screenshotter.js --seleniumIP localhost -b $CIRCLE_JOB --verify --diff --new
```

Comment:
How did this work?

Reply:
We could probably add an alias for the old form, but given that the screenshotter is internal to developers, I doubt it will be too much trouble.